src/monero_ux_nbgl_menu.c

```diff
@@ -117,13 +96,22 @@ static bool account_settings_navigation_cb(uint8_t page, nbgl_pageContent_t* con
         content->choicesList.names = accountNames;
         content->choicesList.token = ACCOUNT_CHOICE;
     } else if (page == 1) {
-        if (N_monero_pstate->account_id > 4) {
-            content->choicesList.initChoice = N_monero_pstate->account_id - 5;
+        if (N_monero_pstate->account_id > 3) {
```

Comment:
Does it mean the elements are hardcoded to be on page1 or page2?
Couldn't we have something more dynamique with nbglv2?

Reply:
That's what I initially tried to do, but unfortunately, it's not possible for multi-tiered settings.